web/src/main/scala/net/wiringbits/components/widgets/ResendVerifyEmailForm.scala

```diff
@@ -16,21 +16,21 @@ import org.scalajs.dom
 import org.scalajs.dom.URLSearchParams
 import org.scalajs.macrotaskexecutor.MacrotaskExecutor.Implicits.global
 import slinky.core.{FunctionalComponent, SyntheticEvent}
-import slinky.core.annotations.react
 import slinky.core.facade.Hooks
 import slinky.web.html._
 import typings.reactRouterDom.mod.useLocation
+
 import typings.reactRouterDom.{mod => reactRouterDom}
 
 import scala.util.{Failure, Success}
 
-@react object ResendVerifyEmailForm {
+object ResendVerifyEmailForm {
   case class Props(ctx: AppContext)
 
   val component: FunctionalComponent[Props] = FunctionalComponent[Props] { props =>
     val texts = I18nHooks.useMessages(props.ctx.$lang)
     val history = reactRouterDom.useHistory()
-    val params = new URLSearchParams(useLocation().search)
+    val params = URLSearchParams(dom.window.location.search)
```

Comment:
This won't likely work but I don't remember what's the exact problem this will cause